### DISCUSSION TITLE
Colon alignment (:)

### DISCUSSION THREAD

**Discussion by Justinzobel** [mikefarah/yq#1945] [Q&A]
Thanks to community help I solved #1926 however when I enter the contents in using this, all of the `:` in my yml file get aligned to the left instead of staying formatted how I like them.

It changes from my preferred formatting of: 
```
name       : test
version    : 1.23
release    : 93
```

into:
```
name: test
version: 1.23
release: 93
```
How can I keep my preferred `:` alignment?

**Comment by mikefarah** [comment]
The YAML encoder I'm using unfortunately doesn't support that formatting

**Reply by roke-julian-lockhart** [reply]
@mikefarah, I'd have liked to utilise `yq` to work around [`rpm-software-management/rpm/discussions/3912#discussioncomment-14284741`](https://github.com/rpm-software-management/rpm/discussions/3912#discussioncomment-14284741), until it's released…
